index.Rmd

```diff
@@ -3,7 +3,9 @@ title: |
   ![](images/DaSEH_banner_wide_transparent.png)
   <br>
 output:
-  html_document
+  html_document:
+    includes:
+      after_body: footer.html
```

Comment:
Would it look weird if this was the footer on every page?

Reply:
actually I was trying to do that - hang tight